### PR TITLE
Feature/update tracing

### PR DESCRIPTION
**Pull request by RBlafford**

### CODE REVIEW

File: source/metrics.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  the time it takes for a message to be fully processed after leaving our
+  incoming queue.
+
+.. code-block:: bash
+
+   [2015-10-23 11:08:08.318] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5141us, p95: 7353us, p99: 9239us, p999: 17194us
+   [2015-10-23 11:08:08.330] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5142us, p95: 7355us, p99: 9238us, p999: 17193us
+   [2015-10-23 11:08:08.344] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5143us, p95: 7370us, p99: 9251us, p999: 17191us
+   [2015-10-23 11:08:08.358] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5144us, p95: 7384us, p99: 9249us, p999: 17189us
+   [2015-10-23 11:08:08.373] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5145us, p95: 7399us, p99: 9381us, p999: 17188us
```
Comment (senior7515) [inline source/metrics.rst]: Explain what does p99 and p999 means etc.

Comment (senior7515) [inline source/metrics.rst]: err nvm.

Comment (senior7515) [inline source/metrics.rst]: Do mention that zipkin integration will be here soon. So all you have to do is point it to your zipkin installation and collect the latencies through time if you are interested.

Comment (senior7515) [inline source/metrics.rst]: or interested in end-to-end latencies.

File: source/metrics.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+================
+Built-In Metrics
+================
+By default each Concord operator logs its performance metrics to a local file
```
Comment (senior7515) [inline source/metrics.rst]: local files? plural .

File: source/metrics.rst (hunk 1 above)
Comment (senior7515) [inline source/metrics.rst]: yeah and zipkin should be it's own section. 

like this 

```
Zipkin
==========

We are in the midst of integrating zipkin latency metrics. Please check back soon to learn how to integrate the zipkin UI <screenshot> into your project. 

We're integrating with the zipkin kafka collector. 

```

**Comment by senior7515** [comment]
LGTM. just some minor points to take a look.

File: source/metrics.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
+================
+Built-In Metrics
+================
+By default each Concord operator logs its performance metrics to local files
```
Comment (bigs) [inline source/metrics.rst]: local files. <-- end sentence

File: source/metrics.rst [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+for viewing detailed information about point to point latency and throughput
```
Comment (bigs) [inline source/metrics.rst]: These files make point to point latency and throughput metrics accessible through the Mesos UI.

File: source/metrics.rst [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+metrics. To view these metrics you can use the Mesos UI. Click on the `sandbox`
```
Comment (bigs) [inline source/metrics.rst]: ~~To view these metrics you can use the Mesos UI~~

Comment (bigs) [inline source/metrics.rst]: To do this, click on the...

File: source/metrics.rst [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+You should see something like this:
+
+.. image:: images/concord-files.png
+
+Latency Metrics
+================
+Concord records the total average latency of sampled traces in microseconds.
+Every time a sampled trace is processed its timestamp is added to a histogram
+and new latency values are calculated and logged. These values include current
+average, 95th percentile, 99th percentile and 999th percentile. The values
```
Comment (bigs) [inline source/metrics.rst]: 99.9th percentile

File: source/metrics.rst [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+inside of each file ending in `*_latencies.txt` are described below.
+
+- dispatcher_latencies.txt: Measurements will report latency metrics
+  representing the time it takes for a message to be fully processed when
+  attempting to send data downstream.
+- principal_latencies.txt: Measurements will report latency metrics representing
+  the time it takes for a message to be fully processed after leaving our
```
Comment (bigs) [inline source/metrics.rst]: the time it takes for the user code to process a message after leaving our incoming queue.

File: source/metrics.rst [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+   [2015-10-23 11:08:08.330] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5142us, p95: 7355us, p99: 9238us, p999: 17193us
+   [2015-10-23 11:08:08.344] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5143us, p95: 7370us, p99: 9251us, p999: 17191us
+   [2015-10-23 11:08:08.358] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5144us, p95: 7384us, p99: 9249us, p999: 17189us
+   [2015-10-23 11:08:08.373] [principal_latencies] [info] 127.0.0.1:31003: traceId: 0, parentId: 0, id: 0, p50: 5145us, p95: 7399us, p99: 9381us, p999: 17188us
+
+
+Throughput Metrics
+==================
+Concord records the total number of messages entering and/or leaving each
+computation in one second intervals. Every second we log how many messages
```
Comment (bigs) [inline source/metrics.rst]: delete second sentence

**Pull request merged**